test/utils/addressFactory.js

```diff
@@ -1,12 +1,22 @@
 function addressFactory(quantity) {
   const addresses = [];
   for (let i = 0; i < quantity; i++) {
-    addresses.push({
-      buildingNumber: `${i} house`,
+    const address = {
       thoroughfareName: `${i} street`,
       postTown: `${i} town`,
       postcode: `${i} code`,
-    });
+    };
+
```

Comment:
When generating the addresses for the tests in the address factory use, 1 in 3 should go to each type of address.